examples/fscloud/variables.tf

```diff
@@ -0,0 +1,59 @@
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The IBM Cloud API token this account authenticates to"
+  sensitive   = true
+}
+
+variable "prefix" {
+  type        = string
+  description = "Prefix for sm instance"
+  default     = "secrets-manager-test"
+}
+
+variable "region" {
+  type        = string
+  description = "Region where resources will be created"
+  default     = "us-east"
+}
+
+variable "existing_resource_group" {
+  type        = bool
+  description = "Whether to use an existing resource group."
+  default     = false
+}
+
+variable "resource_group_name" {
+  type        = string
+  description = "A resource group name to use for this example, if `existing_resource_group` is false a new resource group will be created"
+  default     = "sm-fscloud"
+}
+
+variable "resource_tags" {
+  type        = list(string)
+  description = "Optional list of tags to be added to created resources"
+  default     = []
+}
+
+##############################################################################
+# Event Notification (EN)
+##############################################################################
+
+variable "en_region" {
+  type        = string
+  description = "Region where event notification will be created"
+  default     = "au-syd"
+}
```

Comment:
No need to have `en_region` variable - just have the example use same region for all resources

Reply:
removed